app/pages/application-details.tsx

```diff
@@ -48,19 +53,30 @@ class ApplicationDetails extends Component<Props> {
     const {query} = this.props;
     const {session} = query || {};
     return (
-      <DefaultLayout session={session}>
+      <DefaultLayout session={session} width="wide">
         <ApplicationStatusContainer
           query={query}
           applicationId={this.props.router.query.applicationId}
         />
         <hr />
-
-        <ApplicationDetail query={query} />
-        <IncentiveCalculatorContainer
-          query={query}
-          bcghgid={this.props.router.query.bcghgid}
-          reportingYear={this.props.router.query.reportingYear}
-        />
+        <Row className="application-container">
+          <Col md={8} className="application-body">
+            <ApplicationDetail isAnalyst query={query} />
```

Comment:
See comment on passing a function to render the review component instead of using `isAnalyst`

Reply:
I thought we agreed this is something we want to do in the future instead?

Reply:
Ok, can you add a `TODO` comment then?